### PR TITLE
[SPARK-40557][CONNECT][FOLLOW-UP] Sync python generated proto files

### PR DESCRIPTION
**Pull request by amaliujia**
### What changes were proposed in this pull request?

This PR syncs python generated proto files. The proto changes in this file is generated by https://github.com/apache/spark/blob/master/connector/connect/dev/generate_protos.sh.

### Why are the changes needed?

Python client side proto files are out of sync. Other python related PRs needs to re-generate proto files which has caused troubles on code review. 

We are looking for ways to automatically keep the python proto files in sync. Before that is done, we need to manually update the proto files. 

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

UT

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @zhengruifeng

**Review by zhengruifeng — APPROVED** [review]
LGTM pending CI

File: python/pyspark/sql/connect/plan.py [hunk 1]
```diff
@@ -319,9 +319,9 @@ def __init__(
 
     def _convert_measure(
         self, m: MeasureType, session: Optional["RemoteSparkSession"]
-    ) -> proto.Aggregate.Measure:
+    ) -> proto.Aggregate.AggregateFunction:
         exp, fun = m
-        measure = proto.Aggregate.Measure()
+        measure = proto.Aggregate.AggregateFunction()
```
Comment (amaliujia) [inline python/pyspark/sql/connect/plan.py:324]: This is the only code change that is caused by the proto change. We are lacking of aggregate testing coverage thus there is no tests affected. I will follow up on the testing coverage but that is a different problem.

cc @zhengruifeng

**Comment by zhengruifeng** [comment]
Merged into master

**Pull request closed without merging**